keel-core/src/main/kotlin/com/netflix/spinnaker/keel/exceptions/InvalidConstraintException.kt

```diff
@@ -0,0 +1,7 @@
+package com.netflix.spinnaker.keel.exceptions
+
+import java.lang.RuntimeException
+
+class InvalidConstraintException(
+  message: String
```

Comment:
Probably should add constraint name here, so we know where the error is coming from?
```
class InvalidConstraintException(
  constraintName: String,
  message: String
) : RuntimeException("$constraintName: $message")
``` or something?